### DISCUSSION TITLE
Ubuntu Vitis VM開發機快照點後開機出現問題

### DISCUSSION THREAD

**Discussion by TINGxTING** [bol-edu/HLS-SOC-Discussions#70] [Q&A]
您好~
我按照使用手冊一步一步進行，但在還原快照後啟動，發生了下圖的問題，顯示說記憶體不能為read。
也簡單的按照網路的做法，重新註冊過DLL檔。
錯誤代碼為:E_FAIL (0x80004005)

想請問大家有遇到這種狀況嗎
感謝~
E_FAIL (0x80004005)

![螢幕擷取畫面 2023-09-25 152739](https://github.com/bol-edu/HLS-SOC-Discussions/assets/101039097/d17de3aa-3384-4eef-802d-9cd26e71b10b)

**Comment by kevinjantw** [comment]
有先重開機後再試試? 還原時要用較多記憶體
(或是Virtualbox及VM都刪除再重新安裝)
建議實體電腦至少有16GB以上

**Reply by TINGxTING** [reply]
有開機重試過了，也有刪掉重用過!
然後我的記憶體是16G應該足夠。
想請問快照預設的10G記憶體可以更改嗎?還是其大小是固定的

**Comment by boledulab** [comment]
可以更改但不建議 因為之後的Xilinx Vivado實驗至少需要10GB
你可以請課堂TA協助
